packages/components/src/RadarChart/RadarChart.stories.js

```diff
@@ -35,6 +36,10 @@ const onClick = function () {
 	document.getElementById('clickMessage').innerHTML =
 		'Good job! You clicked ' + event.target.innerHTML + '.';
 };
+const onClickDot = function () {
```

Comment:
Yo should use `action` from `@storybook/addon-actions` instead.

Reply:
Click events now use Storybook actions.